internal/builders/bazel/action.yml

```diff
@@ -51,10 +51,24 @@ runs:
       id: bazelisk
       uses: bazelbuild/setup-bazelisk@95c9bf48d0c570bb3e28e57108f3450cd67c1a44 # v2.0.0
 
+    - name: Setup Java
+      id: java
+      uses: actions/setup-java@5ffc13f4174014e2d4d4572b3d74c3fa61aeb2c2 # v3.11.0
+      with:
+        distribution: "${{ fromJson(inputs.slsa-workflow-inputs).user-java-distribution }}"
+        java-version: "${{ fromJson(inputs.slsa-workflow-inputs).user-java-version }}"
+
+    - name: Set Env Var $JAVA_HOME
+      shell: bash
+      run: |
+        export JAVA_HOME="$JAVA_HOME"
```

Comment:
What does this do? Isn't `JAVA_HOME` set by `setup-java`?

Reply:
This was supposed to export JAVA_HOME manually so that it could be used to make the local Java repository rule within the WORKSPACE file. I did it to solve this issue:
https://github.com/actions/runner/blob/master/docs/adrs/0278-env-context.md#dont-populate-the-env-context-with-environment-variables-from-runner-machine

However, I just tested it, and it is not the case anymore that it must be manually exported, so this step is actually un-needed. Thanks!